README.md

```diff
@@ -133,6 +133,44 @@ cd Binaries
 
 Please note the current build script is only tested on Ubuntu 14.04. Please update the script accordingly for other distributions.
 
+## Getting Started on OSX
+
+*Note that OSX is not officially supported yet, but it has been seen to work, YMMV. This build has been tested to work when the dependencies are installed via homebrew, YMMV if using FINK or MacPorts*
+
+Install dependecies with homebrew:
+
+```
+brew install glibmm libxml++ libsigc++ ossp-uuid gettext openssl
+```
+
+As mentioned above, the Azure Storage Client Library for C++ depends on Casablanca. Follow [these instructions](https://github.com/Microsoft/cpprestsdk/wiki/How-to-build-for-Mac-OS-X) to compile it. Current version of the library depends on Casablanca version 2.8.0.
+
```

Comment:
It seems the latest (v2.8) `cpprestsdk` is already in homebrew.  It can be installled via `brew install cpprestsdk`.